### DISCUSSION TITLE
Better score for worst performance

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#24974] [Q&A]
I got `S` with `DT 2.0x` he got `A` with `DT 1.5x` yet he has a better score

My guess is that score system changed but the scores haven't been recalculated when it changed?

![image](https://github.com/ppy/osu/assets/40531871/e81a803b-a694-46d5-8bfa-f4ae648c1661)

**Comment by peppy** [comment]
We're not fixing scores until we do a reset later this year.
